portfolio.css

```diff
@@ -0,0 +1,110 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap");
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin-bottom: 2px;
+  margin-top: auto;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: #e5e5e5;
+  font-family: "Poppins", sans-serif;
+}
+
+.first {
+  width: 375px;
```

Comment:
> Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels

- [OPTIONAL] I am going to make optional because it wasn't pointed out by the previous reviewer, According to the project requirement, It suggests you use responsive values like `percentage` over `pixels` for your widths, I'll suggest you update it here, using responsive values makes your project adapt easily to different not mobile screen sizes and still looks great. :smile: :100: